packages/models-library/src/models_library/services.py

```diff
@@ -480,25 +480,104 @@ def validate_thumbnail(cls, value):  # pylint: disable=no-self-argument,no-self-
 
 
+_EXAMPLE = {
+    "name": "oSparc Python Runner",
+    "key": "simcore/services/comp/osparc-python-runner",
+    "type": "computational",
+    "integration-version": "1.0.0",
+    "progress_regexp": "^(?:\\[?PROGRESS\\]?:?)?\\s*(?P<value>[0-1]?\\.\\d+|\\d+\\s*(?P<percent_sign>%))",
+    "version": "1.7.0",
+    "description": "oSparc Python Runner",
+    "contact": "smith@company.com",
+    "authors": [
+        {
+            "name": "John Smith",
+            "email": "smith@company.com",
+            "affiliation": "Company",
+        },
+        {
+            "name": "Richard Brown",
+            "email": "brown@uni.edu",
+            "affiliation": "University",
+        },
+    ],
+    "inputs": {
+        "input_1": {
+            "displayOrder": 1,
+            "label": "Input data",
+            "description": "Any code, requirements or data file",
+            "type": "data:*/*",
+        }
+    },
+    "outputs": {
+        "output_1": {
+            "displayOrder": 1,
+            "label": "Output data",
+            "description": "All data produced by the script is zipped as output_data.zip",
+            "type": "data:*/*",
+            "fileToKeyMap": {"output_data.zip": "output_1"},
+        }
+    },
+}
+
+_EXAMPLE_W_BOOT_OPTIONS_AND_NO_DISPLAY_ORDER = {
+    **_EXAMPLE,
+    "description": "oSparc Python Runner with boot options",
+    "inputs": {
+        "input_1": {
+            "label": "Input data",
+            "description": "Any code, requirements or data file",
+            "type": "data:*/*",
+        }
+    },
+    "outputs": {
+        "output_1": {
+            "label": "Output data",
+            "description": "All data produced by the script is zipped as output_data.zip",
+            "type": "data:*/*",
+            "fileToKeyMap": {"output_data.zip": "output_1"},
+        }
+    },
+    "boot-options": {
+        "example_service_defined_boot_mode": BootOption.Config.schema_extra["examples"][
+            0
+        ],
+        "example_service_defined_theme_selection": BootOption.Config.schema_extra[
+            "examples"
+        ][1],
+    },
+    "min-visible-inputs": 2,
+}
+
+
 class ServiceDockerData(ServiceKeyVersion, _BaseServiceCommonDataModel):
     """
     Static metadata for a service injected in the image labels
 
-    This is one to one with node-meta-v0.0.1.json
+    NOTE: This model serialized in .osparc/metadata.yml and in the labels of the docker image
     """
 
+    version_display: str | None = Field(
+        None,
+        description="A user-friendly or marketing name for the release."
+        " This can be used to reference the release in a more readable and recognizable format, such as 'Matterhorn Release,' 'Spring Update,' or 'Holiday Edition.'"
+        " This name is not used for version comparison but is useful for communication and documentation purposes.",
+    )
+
+    release_date: date | None = Field(
```

Comment:
Q: we do not need to use a datetime here? you are sure that every release will have at least 1 day of difference? otherwise asked, we do not plan to sort via this entry?